shared/js/background/redirect.es6.js

```diff
@@ -43,12 +43,14 @@ function buildResponse (url, requestData, tab, isMainFrame) {
 
 function handleRequest (requestData) {
     const tabId = requestData.tabId
-    const blockingEnabled = utils.isFeatureEnabled('contentBlocking')
+    let blockingEnabled = utils.isFeatureEnabled('contentBlocking')
```

Comment:
This also should be an `&&` right?
```suggestion
    let blockingEnabled = utils.isFeatureEnabled('contentBlocking') && !utils.isUrlExcludedByFeature(thisTab.url, 'contentBlocking')
```

Alternatively the single function could make both checks?